news-buddy/src/components/SingleArticle.jsx

```diff
@@ -3,18 +3,42 @@ import { getArticleById } from "../api"
 import { useState, useEffect } from "react"
 import Loading from "./Loading"
 import CommentsList from "./CommentsList"
+import { updateVotes } from "../api"
 
 function SingleArticle() {
     const {article_id} = useParams()
     const [isLoading, setIsLoading] = useState(true)
     const [article, setArticle] = useState({})
-    const [newVotes, setNewVotes] = useState(article.votes)
+    const [votes, setVotes] = useState(0)
+
+    function handleLike(){
+        if(votes === 0){
```

Comment:
this if condition means that we will only ever be able to vote when the vote is 0 - we wont be able to vote in any other scenario... so we dont need this if condition at all! :) Same for the dislike